syncd/VendorSai.cpp

```diff
@@ -661,6 +661,7 @@ sai_status_t VendorSai::bulkGetStats(
     SWSS_LOG_ENTER();
     VENDOR_CHECK_API_INITIALIZED();
 
+#ifdef HAVE_SAI_BULK_OBJECT_GET_STATS
     return sai_bulk_object_get_stats(
```

Comment:
are `sai_bulk_object_get_stats` and `sai_bulk_object_clear_stats` defined in vendor SAI?
If yes, the fix should resolve the compiling error.
Otherwise, we may want to find out the place where they defined and exclude the code which calls vendor SAI.